src/NServiceBus.Core/obsoletes.cs

```diff
@@ -739,6 +739,15 @@ public partial class Feature
             throw new NotImplementedException();
         }
     }
+
+    [ObsoleteEx(
+        Message = "Use extensions provided by the TransportDefinition class instead",
```

Comment:
`Message` and `ReplacementTypeOrMember` are usually not meant to be combined. Since in this case its not a straight conversion I'd say just keep the `Message` ?